### PR TITLE
Us99323/hm initials

### PR DESCRIPTION
**Pull request by mtjandra**
Hyper-media support initials

### CODE REVIEW

File: test/d2l-profile-image-hm.html [hunk 1]
```diff
@@ -43,6 +43,69 @@
...
+						.returns(Promise.resolve(window.D2L.Hypermedia.Siren.Parse(response)));
+
+					defaultElement.href = href;
+
+					defaultElement._fetchUserInformation()
+						.then(function() {
+							flush(function() {
+								assert.equal(defaultElement._firstName, t.anonFirst ? '' : firstName);
+								assert.equal(defaultElement._lastName, t.anonLast ? '' : lastName);
+								assert.equal(defaultElement._colourId, userId);
```
Comment (jeroach) [inline test/d2l-profile-image-hm.html]: This might just be a simplification for the test, but it seems odd that the `colourId` would match the `userId`

Comment (mtjandra) [reply test/d2l-profile-image-hm.html]: The hyper-media api doesn't provide a color ID. That might be something we want to do. When I spoke with Bob the user's color ID seems to be local to this component, so I was actually thinking of using the userId as the colourId

File: d2l-profile-image-hm.html [hunk 2]
```diff
@@ -53,9 +68,32 @@
 			},
 			_fetchUserInformation: function() {
 				return this._fetchEntity(this.href)
-					.then(function() {
-						// TODO: Set the attributes on the profile image
-					})
+					.then(function(data) {
+						var firstName = data.getSubEntityByRel('https://api.brightspace.com/rels/first-name');
+						if (!firstName.hasClass('default-name')) {
+							this.set('_firstName', firstName.properties.name);
```
[the remark this answers is not in the document]
Comment (mtjandra) [reply d2l-profile-image-hm.html]: If the name is `null`, the than `_firstName` will be set to null and the base component should be able to handle that

File: test/d2l-profile-image-hm.html (hunk 1 above)
Comment (jeroach) [reply test/d2l-profile-image-hm.html]: Sounds good. I'll approve what's here - if we decide to add the colour ID to the API, I assume that will be a follow up PR.

**Pull request merged**